src/queries/sessions/sessions-api.js

```diff
@@ -9,3 +9,15 @@ export const createSession = async (body) => {
 
   return response.json();
 };
+
+export const getAllSessions = async () => {
+  const response = await fetch(`http://${url}/v1/sessions`, { method: "GET" });
+  return response.json();
+};
+
+export const getSession = async ({ sessionId }) => {
+  const response = await fetch(`http://${url}/v1/sessions/${sessionId}`, {
```

Comment:
http can be removed since @caseyyee 's change to the .env file

Reply:
will merge and fix.  avery is heading out for tonight.

Reply:
Sounds good. I was just about to say I have time to burn here :)